bin/schema_generator.sh

```diff
@@ -92,9 +94,12 @@ function validated_generate_commit() {
 
     # Generate schemas, commits results to the publish branch
     generate_commit "$mps_root" "$mps_branch_source" "$mps_branch_publish"
-    validate-bigquery local-validation \
-        --head "$mps_branch_publish" \
-        --base "$mps_branch_temp"
+
+    if [ "$MPS_VALIDATE_BQ" != "false" ]; then
```

Comment:
It may also be worth considering `set +e` before and `set -e` after during the script based on the environment variable.

Reply:
The implications of that are not completely clear to me, so I'm going to go ahead and merge this as-is (since it's relevant to concurrent work). But feel free to file a follow-up.